src/sdk.js

```diff
@@ -0,0 +1,72 @@
+import Web3 from 'web3';
+import IPFSClient from 'ipfs-http-client';
+import url from 'url';
+import RegistryNetworks from 'singularitynet-platform-contracts/networks/Registry.json';
+import RegistryAbi from 'singularitynet-platform-contracts/abi/Registry.json';
+
+import Account from './Account';
+import MPEContract from './MPEContract';
+import ServiceClient from './ServiceClient';
+import { find } from 'lodash';
+
+const DEFAULT_GAS_LIMIT = 210000;
+const DEFAULT_GAS_PRICE = 4700000;
+
+export default class SnetSDK {
+  constructor(config) {
+    this._config = config;
+    const options = {
+      defaultGas: this._config.defaultGasLimit || DEFAULT_GAS_LIMIT,
+      defaultGasPrice: this._config.defaultGasPrice || DEFAULT_GAS_PRICE,
+    };
+    this._networkId = config.networkId;
+    this._web3 = new Web3(config.web3Provider, null, options);
+    this._mpeContract = new MPEContract(this._web3, this._networkId);
+    this._account = new Account(this._web3, this._networkId, this._config, this._mpeContract);
+    this._web3.eth.defaultAccount = this._account.address;
+    const registryAddress = RegistryNetworks[this._networkId].address;
+    this._registryContract = new this._web3.eth.Contract(RegistryAbi, registryAddress, { from: this._account.address });
+  }
+
+  get account() {
+    return this._account;
+  }
+
+  get mpeContract() {
```

Comment:
I dont' think we need to provide `mpeContract` as part of public SDK API it makes API ambiguous: you can make the same thing using different approaches.

Reply:
So in general there are a bunch of methods exposed so that the developer can have access to them when they are building their own `PaymentChannelManagementStrategy`.
Listing down why each of them could be useful for someone developing custom `PaymentChannelManagementStrategy`
1. `MPEContract` -> This would be useful to open/fund/extend payment channels in the strategy. 
2. `web3` -> For getting the current block number and other web3 related utility methods which would come in handy. 
3. `Account` -> To perform account related operations like approving funds, transferring funds etc...

These instances expose multiple building building blocks which would enable developers to easily build simple and complex strategies. Otherwise they would have to redo these generic operations themselves.

Reply:
When I say "ambiguous" I mean that `Account` and `ServiceClient` as a whole provides duplicate of `MPEContract` interface. My first impression is that you add `Account` and `ServiceClient` to allow client manipulate balance and channel and these classes use `web3` and `MPEContract` as backend. It is not clear for me if user can use `MPEContract` to do all things why `Account` and `ServiceClient` duplicates its API?

Reply:
Agree with removing `MPEContract` from the public API. Have addressed that in [commit](https://github.com/singnet/snet-sdk-js/pull/1/commits/78cf9ae1c7ce7ce7b1812ea3d016dc3691663314). However I feel we should expose `web3` since there could be a bunch of utility functions from web3 which will be handy.